src/System.ComponentModel.TypeConverter/src/Resources/System.ComponentModel.TypeConverter.rd.xml

```diff
@@ -13,6 +13,9 @@
         </Type>
         <Type Name="TypeConverterAttribute">
           <AttributeImplies Activate="Required Public" />
+          
+          <!-- AttributeCollection.GetDefaultAttribute will end up reflecting on the Default field -->
```

Comment:
This is OK, but should we have an intrinsic or something else that should pick this up?